drivers/windows/windows.go

```diff
@@ -457,20 +471,21 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		VirtualNetwork: n.config.HnsID,
 	}
 
-	ec, err := parseEndpointOptions(epOptions)
+	epOption, err := parseEndpointOptions(epOptions)
+	epConnectivity, err := parseEndpointConnectivity(epOptions)
```

Comment:
Thanks. I think this is fine for now,  it is done internally in libnetwork and no docker/docker changes should be needed to accommodate this. Good to hear you guys are planning to support it in future.